packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -213,7 +214,20 @@ module.exports = {
     // to restart the development server for Webpack to discover it. This plugin
     // makes the discovery automatic so you don't have to restart.
     // See https://github.com/facebookincubator/create-react-app/issues/186
-    new WatchMissingNodeModulesPlugin(paths.appNodeModules)
+    new WatchMissingNodeModulesPlugin(paths.appNodeModules),
+    // Trigger some typechecking if a file matches with an @ flow comment
+    new FlowTypecheckPlugin({
+      flowconfig: [
+        '[ignore]',
+        '<PROJECT_ROOT>/node_modules/fbjs/.*',
+        '',
+        '[libs]',
+        './flow-typed'
```

Comment:
Hmm, this should work. For example, if you do

```
$ mkdir foo && cd foo
$ flow init
$ flow-typed install --flowVersion=0.36.0 jest@17
$ echo -e "// @flow\n\n it('does stuff', () => {});" > foo.test.js
$ flow
No errors!
$ rm -rf flow-typed/
$ flow
foo.test.js:3
  3:  it('does stuff', () => {});
      ^^ identifier `it`. Could not resolve name
```

I'm not exactly sure how to test this PR. I tried the following:

```
git remote add rricard git@github.com:rricard/create-react-app.git
g fetch rricard
g co rricard/integrate-flow
yarn install
yarn create-react-app foo/
cd foo
npm3 run start
```

and then add `// @flow` to `src/App.test.js`. However, modifying that file doesn't seem to trigger a rebuild. And then when I modify `src/App.js`, it DOES run Flow, but it doesn't generate a `.flowconfig` or run `flow-typed`. So I'm probably doing something wrong. I can confirm that `node_modules/react-dev-utils/FlowTypecheckPlugin.js` does exist though!